lib/engines/content_block_manager/app/components/content_block_manager/content_block/document/index/filter_options_component.rb

```diff
@@ -11,18 +11,30 @@ def items_for_block_type
       {
         label: schema_name.humanize,
         value: schema_name,
-        checked: !@filters.nil? && @filters[:block_type]&.include?(schema_name),
+        checked: @filters.any? && @filters[:block_type]&.include?(schema_name),
       }
     end
   end
 
-  def options_for_lead_organisation
+  def all_organisations_option
+    {
+      text: "All organisations",
+      value: "",
+      selected: @filters.none? || @filters[:lead_organisation]&.empty?,
+    }
+  end
+
+  def taggable_organisations_options
     helpers.taggable_organisations_container.map do |name, id|
       {
         text: name,
         value: id,
-        selected: !@filters.nil? && @filters[:lead_organisation] == id.to_s,
+        selected: @filters.any? && @filters[:lead_organisation] == id.to_s,
       }
     end
   end
+
+  def options_for_lead_organisation
+    [all_organisations_option].push(*taggable_organisations_options)
```

Comment:
Would `[all_organisations_option, taggable_organisations_options].flatten` be more readable here?